src/libmongoc/doc/logging.rst

```diff
@@ -100,6 +100,9 @@ For example, you could register a custom handler to suppress messages at INFO le
      return 0;
   }
 
+Note that in the example above `mongoc_log_set_handler()` is called before ``mongoc_init()``.
```

Comment:
Here `mongoc_log_set_handler()` needs two backquote keys around it for monospaced fonts to work.